Makefile

```diff
@@ -50,6 +50,9 @@ test: codegen download-test-deps
 	KUBEBUILDER_ASSETS="$(shell setup-envtest --use-env use -p path)" \
 		ginkgo -nodes=${TEST_NODES} --randomize-all ./test/...
 
+lint:
+	golangci-lint run --timeout 10m --build-tags integration
```

Comment:
I don't think we should be using a build tag to segregate tests (see my comment in #387), so `--build-tags integration` should be redundant here.